lib/plugins/aws/deploy/lib/updateStack.js

```diff
@@ -49,6 +49,7 @@ module.exports = {
     return new BbPromise((resolve, reject) => {
       let stackStatus = null;
       let stackData = null;
+      let count = 0;
```

Comment:
Can we move this variable in the loading method so that it's encapsulated? Otherwise users might miss it and the loading won't work correctly.